app/views/application/search/add-details.html

```diff
@@ -4,83 +4,65 @@
 {% set formAction = "./review" %}
 
 {% set captionHeading %}
-  {# If they are coming through the Sector flow #}
-  {% if data.referrer == "sectorSearch" %}
-    {{ data.selectedIndustry }}
-    {% if data.selectedQualificationSector == "I don't know the qualification type" %}
-      {% if data.selectedLevelSector == "I don't know the level" %}
+  {# Coming from the assessment only, qual only route #}
+  {% if data.referrer == "assessmentExpertise" %}
+    {% if data.selectedQualification == "I don't know the qualification type" %}
+      {% if data.selectedLevelAssessmntOnly == "I don't know the level" %}
         {% else %} 
-        ({{ data.selectedLevelSector }})
+        ({{ data.selectedLevelAssessmntOnly }})
```

Comment:
Just checking the spelling of 'assessment' here. It is missing an 'e'.

Reply:
Good spot - I've gone through and fixed this throughout